cmake/projects/NASM/schemes/url_sha1_nasm_unpack_windows.cmake.in

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2017, Zhuhao Wang
```

Comment:
So this one is the same as https://github.com/ruslo/hunter/blob/master/cmake/schemes/url_sha1_unpack.cmake.in? Also I think it should be https://github.com/ruslo/hunter/blob/master/cmake/schemes/url_sha1_unpack_install.cmake.in